infra/api/service/copy_oracle_data.tf

```diff
@@ -0,0 +1,89 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sfn_state_machine
+resource "aws_sfn_state_machine" "copy_oracle_data" {
```

Comment:
Named every resource `copy_oracle_data` in anticipation of more state machines in the future